lib/worker/dataset.js

```diff
@@ -10,7 +10,7 @@
 const { getOrNotFound } = require('../util/promise');
 
 const createEntitiesFromPendingSubmissions = async ({ Entities, Datasets }, event) => {
-  if (event.details.data && !event.details.data.approvalRequired && event.details.autoConvert) {
+  if (event.details.data && event.details.data.approvalRequired === false && event.details.autoConvert) {
```

Comment:
This change isn't strictly necessary, but I feel like it's more explicit. We only want to do the following if `approvalRequired` is `false`, not if it's `undefined`.